app/src/main/res/drawable/background_phrase_border.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Filename hints where this file was copied from 😄Don't forget to change names to make it clearer where they are used in current app.